src/schema/__tests__/auction_results.test.js

```diff
@@ -92,9 +92,9 @@ describe("Artist type", () => {
                   },
                   currency: "JPY",
                   price_realized: {
-                    cents: 420,
-                    cents_usd: 100,
-                    display: "¥ JPY420",
+                    cents: 420000,
+                    cents_usd: 100000,
+                    display: "¥ JPY420k",
```

Comment:
I'd expect "JPY ¥420k" here (swapping symbol and code locations), but am open to this if that's what the design calls for.

Reply:
Oh yea that's what I meant! the comp just included a euro example, I just swapped symbol/code here.